app/client/public/logger.js

```diff
@@ -0,0 +1,81 @@
+// Gets the current Unix timestamp
+function getCurrentUTCTimestamp(date) {
+  return Math.floor((date || new Date()).getTime() / 1000);
+}
+
+// Gets the unix timestamp of the hour
+// For a current time of 19:15, returns the timestamp of 19:00
+function getCurrentUTCHourTimestamp() {
+  const date = new Date();
+  const timestamp = getCurrentUTCTimestamp(date);
+  return timestamp - (date.getUTCMinutes() * 60 + date.getUTCSeconds());
```

Comment:
I gather from the comments above, that the point  here is to reset minutes and seconds to 0 from current time. Would this be more reliable then?

```suggestion
  date.setUTCMinutes(0);
  date.setUTCSeconds(0);
  return getCurrentUTCTimestamp(date);
```

Reply:
Yes, we can, they both result in the same value.

<img width="445" alt="Screenshot 2022-04-01 at 12 11 45 PM" src="https://user-images.githubusercontent.com/103687/161209633-fb230af8-8146-47ee-9af0-47b62cd8b075.png">

Reply:
Got it. No worries then. Less of an issue than the other comment. :)